test/test-suite/groups/performance/performance.json

```diff
@@ -0,0 +1,132 @@
+{
+    "expr": "{        \"data\": [          {            \"fields\": {              \"lineItems\": (                $rows := $distinct(result[0].payload[label = \"table\"].cells[].row);                $map($rows, function($row) { result[0].payload[label = \"table\"].cells[row = $row] })              )                .{                  \"reason_line_item\": (                    $invoice_number := ($[label = \"invoice_number_line_item\"].text).$lowercase();                    $keys($$.tables.dataTable[String != \"\" and String != \"\" and String != \"\" and String != \"\" and String != \"\" and String != \"\" and String != \"\" and String != \"\"and String = \"Promo\"]) = 0 ? \"foo\" : \"bar\"                  )                }            }          }        ]      }",
```

Comment:
Shame this is a json file, not a JS file – would be great to make this expression more readable

Reply:
Yup! I'm following all the pre-existing conventions here, at least for the time being.